webapp/lib/view.dart

```diff
@@ -549,6 +549,60 @@ class ContentView {
     driverChartsTabContent.children.clear();
   }
 
+  void toggleChartLoadingState(controller.ChartType chartType, bool state, [bool isSystemEvents = false]) {
+    switch(chartType){
+      case controller.ChartType.conversation:
+        if (state) {
+          needsReplyLatestValue.spinner.classes.remove('hidden');
```

Comment:
I would suggest renaming `state` to `show` as it makes it a bit clearer what it means, and then you can use the `toggle()` method to show/hide the `hidden` class based on it, which means that you won't need the if/else statement:
```suggestion
          needsReplyLatestValue.spinner.classes.toggle('hidden', show);
```

Reply:
Improved the logic with this